replication/v1/replication.proto

```diff
@@ -131,6 +131,12 @@ message CloneDeploymentFromBackupRequest {
     // Target project identifier.
     // This is an optional field
     string project_id = 4;
+    // Identifier of disk performance to use for the cloned deployment.
+    // This field is required if the deployment is cloned to a cloud provider
+    // different from the source deployment.
+    // This field is ignored if the Deployment is cloned within the same cloud provider,
+    // i.e, the same disk performance setting is used.
+    string disk_performance_id = 5;
```

Comment:
IMO not needed, we can 'simply' take the default per deployment size (and provider), this is a very advanced property IMO only needed when the custom disk setting bool is set (on org level), but would not at it at all tbh.